homeassistant/components/combined_energy/strings.json

```
@@ -0,0 +1,127 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Define the API parameters for this installation",
+        "data": {
+          "name": "The name of this installation",
+          "username": "Email or Mobile Number",
+          "password": "[%key:common::config_flow::data::password%]",
+          "installation_id": "Combined Energy Installation-ID"
+        }
+      }
+    },
+    "error": {
+      "installation_not_accessible": "Permission denied for Installation ID",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "energy_consumed": {
+        "name": "Energy Consumption"
```

```suggestion
        "name": "Energy consumption"
```

Only the first letter of the first word should be upper case. Please adopt the others